### ISSUE TITLE
[BUG] HoverTool doesn't take on theme of plot

### ISSUE THREAD

**Issue by rchurt** [bokeh/bokeh#10322]
When using `curdoc().theme = 'dark_minimal'` to set the theme of a plot, the theme is not applied to the HoverTool. I'd expect it to have a dark background, in this case.

bokeh 2.1.1
Python 3.6.9
Chrome 84.0.4147.89
Google Colab

<img width="1014" alt="Screen Shot 2020-07-17 at 7 20 21 PM" src="https://user-images.githubusercontent.com/14302923/87842644-b5c29580-c862-11ea-98b8-fcdc71282234.png">

**Comment by bryevdv** [comment]
Right, themes only control the default values of the typed properties of Bokeh models, and there are not currently really any properties available to tweak the default tooltip layout. This will require some thought, as there are a couple of possible avenues:

* Open a mechanism for themes to control CSS settings,  possibly in some prescribed or limited way (@mattpap will surely have thoughts about this)

* Add typed properties (which then be theme-able) to the `HoverTool` to control the appearance of the default "table" tooltip

For the time being, I think your best best is to create a custom HTML tooltip as described in the User's Guide
